src/views/router/routes.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Chat from '../chat/chat'
+import { Home } from '../home'
+import Settings from '../settings/settings'
+import { FaCog, FaComment, FaHome } from 'react-icons/fa'
+
+/* eslint-disable sort-keys */
+export const appRoutes = {
+	Home: {
+		component: Home,
+		icon: <FaHome />,
+		label: 'Home',
+	},
+	Chat: {
+		component: Chat,
+		icon: <FaComment />,
+		label: 'Chat',
+	},
+	Settings: {
+		component: Settings,
+		icon: <FaCog />,
+		label: 'Settings',
```

Comment:
With the other changes, on this file we can remove the HTML brackets, and convert this file back to a `.ts` file.
```suggestion
	Home: {
		component: Home,
		icon: FaHome,
		label: 'Home',
	},
	Chat: {
		component: Chat,
		icon: FaComment,
		label: 'Chat',
	},
	Settings: {
		component: Settings,
		icon: FaCog,
		label: 'Settings',
```